getting_started/first_3d_game/05.spawning_mobs.rst

```diff
@@ -234,20 +238,21 @@ Let's code the mob spawning logic. We're going to:
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-   func _on_MobTimer_timeout():
-       # Create a new instance of the Mob scene.
-       var mob = mob_scene.instantiate()
+   func _on_mob_timer_timeout():
+        # Create a new instance of the Mob scene.
+        var mob = mob_scene.instantiate()
```

Comment:
Indentation inconsistency (3, then 8).

Looks like most code blocks here were wrongly indented with 3 spaces, leading to inconsistencies. This should be reviewed.